### ISSUE TITLE
After applying filter commands, wrapwidth stops working on those lines until toggled

### ISSUE THREAD

**Issue by Jorenar** [rickhowe/wrapwidth#13]
If we filter some lines, or whole file, e.g. with `:'<,'>!cat`, then the effects of `:Wrapwidth` stop working on those lines until whole plugin gets toggled either via `:set wrap!` or `:Wrapwidth 0`.  Reapplying `:Wrapwidth N` or executing `:Wrapwidth!` doesn't have any effect.

---

"Raw":
```
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```
`:set wrap | Wrapwidth 76`:
```
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod 
tempor incididunt ut labore et dolore magna aliqua.
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod 
tempor incididunt ut labore et dolore magna aliqua.
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod 
tempor incididunt ut labore et dolore magna aliqua.
```
`:2!cat`:
```
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod 
tempor incididunt ut labore et dolore magna aliqua.
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod 
tempor incididunt ut labore et dolore magna aliqua.
```

---

Content of `b:wrapwidth`:
* before filter: `{'to': 0, 'lw': {'1': 76, '2': 76, '3': 76}, 'wd': 212, 'ch': [], 'lc': 1}`
* after filter: `{'to': 0, 'lw': {'1': 76, '2': 76, '3': 76}, 'wd': 212, 'ch': [], 'lc': 1}`

**Comment by rickhowe** [comment]
I tried to fix it. Can I ask you to check with this change?

```
@@ -207,6 +207,12 @@
   elseif ev == 'TextChanged' || ev == 'TextChangedI'
     if !has('nvim') | call listener_flush(cb) | endif
     let bw = getbufvar(cb, s:ww)
+    if len(bw.ch) == 2
+      let [s0, e0, n0] = bw.ch[0] | let [s1, e1, n1] = bw.ch[1]
+      if 0 < n0 && 0 > n1 && s1 - s0 == n1
+        let bw.ch =[[s1, e1, n0 + n1]]
+      endif
+    endif
     let ul = []
     for [sl, el, na] in bw.ch
       if na < 0
```

**Comment by Jorenar** [comment]
Seems like a fix :)

**Comment by rickhowe** [comment]
Thanks. I will fix it on the next version.

**Comment by Jorenar** [comment]
Fixed in update 3.4 (c592b47)
